src/components/shared/LanguageSelector/LanguageSelector.js

```diff
@@ -9,29 +9,52 @@ import { injectIntl } from 'react-intl';
  */
 const LanguageSelector = ({ intl }) => {
   const lang = intl.locale;
+  const languageDropdown = useRef(null);
```

Comment:
```suggestion
  const languageDropdownEl = useRef(null);
```

Reply:
```suggestion
  const languageDropdownRef = useRef(null);
```

Reply:
That's a good idea, @andresmoschini . After @cbernat is finish fixing some tests that are currently failing, I'll do that improvement 👍